src/components/modal/content/AU/parts/PL.jsx

```diff
@@ -0,0 +1,44 @@
+/** @jsx h */
+import { h } from 'preact';
+import { useContent, useProductMeta } from '../../../lib';
+
+const PL = () => {
+    const { headline, subHeadline, terms, instructions } = useContent('GPL');
+    const { qualifying: qualifyingString } = useProductMeta('GPL');
+    const qualifying = qualifyingString?.toLowerCase() === 'true';
+
+    return (
+        <section className="content-body">
+            <div className="description">
+                <h2>{headline.singleProduct}</h2>
+
+                <h3>{qualifying ? subHeadline.qualified : subHeadline.unqualified}</h3>
```

Comment:
Regex can be added here - only seem to need it for unqualified since the qualifying subheadline doesn't include the purchase range.
```suggestion
                <h3>
                    {qualifying ? subHeadline.qualified : auCurrencyFormat(subHeadline.unqualified)}
                </h3>
```